src/form.js

```diff
@@ -172,6 +172,40 @@ define([
     }
 
     Form.prototype = {
+        dataTree: function() {
```

Comment:
Seems almost silly to have this after taking out the original data tree, but I like that we can see how data tree relates to the tree in one function. Thought about also providing a helper for control tree.